src/EN/026_create_splashpool.ts

```diff
@@ -0,0 +1,61 @@
+import { AnchorProvider } from "@coral-xyz/anchor";
+import {
+  WhirlpoolContext,
+  buildWhirlpoolClient,
+  ORCA_WHIRLPOOL_PROGRAM_ID,
+} from "@orca-so/whirlpools-sdk";
+import { PublicKey } from "@solana/web3.js";
+import Decimal from "decimal.js";
+
+// UNIX/Linux/Mac
+// bash$ export ANCHOR_PROVIDER_URL=https://api.devnet.solana.com
+// bash$ export ANCHOR_WALLET=wallet.json
+// bash$ ts-node this_script.ts
+//
+// Windows
+// > set ANCHOR_PROVIDER_URL=https://api.devnet.solana.com
+// > set ANCHOR_WALLET=wallet.json
+// > ts-node this_script.ts
+
```

Comment:
Is it possible to add short explanation for "What is SplashPool" here as code comment ?
Points:
- it has only 2 TickArrays (simple, low cost)
- it allows FullRange positions only (similar to Constant Product AMM)
- it is Whirlpool with a specific tick_spacing. SplashPool can be handled as Whirlpool.